src/components/Alert/Alert.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import renderWithRedux from '../../testUtils/renderWithRedux';
+import Alert from './Alert';
+
+describe('Alert', () => {
+  it('shows alert', () => {
+    const { getByTestId } = renderWithRedux(<Alert />, {
+      initialState: {
+        alert: {
+          showAlert: true,
+          type: 'error',
+          message: 'Some error appeared...',
+        },
+      },
+    });
+    const alert = getByTestId('alert');
+
+    expect(alert).toHaveClass('active');
+    expect(alert).toHaveTextContent(/Some error appeared/i);
+  });
+
+  it('hides alert', () => {
+    const { getByTestId } = renderWithRedux(<Alert />);
```

Comment:
In this test you actually doesn't hides a visible alert, but just renders it in initial (hidden) state.

Reply:
solved here https://github.com/maxSedelnikov/react-shopping-app/pull/6/commits/c759e7206dd9006634852446e70ed64047e50a1d